Majasdarbs/06-majasdarbs/GvidoZarins/script.js

```diff
@@ -0,0 +1,66 @@
+/*  
+    Aplikacijas darba principu var apskatities video
+    success-case.mp4
+    bad-case.mp4
+
+    Aplikacijas darba princips:
+        Mums piejajams masivs ar burtiem
+        Lietotajam ir iespeja ievadit skaitli 
+        Ja ievaditais skaitlis atbilst burtu masivam tad šis burts tiek izvadits uz ekranu 
+        Spiežot pogas uz tastaturas aplikacija parbauda ievadito vertibu ar izveleto burtu un pažiņo par rezultatu
+*/
+
+
+// Implementacija
+// 1) izvejdot massivu ar burtiem characters[]
+
+// 2) izvejtot mainigo "arrayId" šis mainigas saņems lietotaja ievadito skaitli
+
+// 3) izvejtot mainigo "output" šis mainigas satur DOM elementu kur tiek izvadits izveletais burts (getElementById) 
+
+/* 4) izvejtot funkciju getValue() funkcijas ipašibas 
+        neko nesaņem
+        izmantojot prompt() medoti --> https://www.w3schools.com/jsref/met_win_prompt.asp piesaistit vertibu mainigam arrayId
+        izveidot parbaudi if else un parliecinaties ka ievietotais skatlis pieder intervalam no 0 lidz characters.lenght
+        ja gadijuma izsaukt finkciju print value (5 punkts)
+        ne gadijuma izvadit kļudas paziņojomu pec tam izsaukt funkciju getValue velreizi
+*/      
+
+/* 5) izvejtot funkciju printValues() funkcijas ipašibas
+        neko nesaņem
+        izmaiņam defineto output mainigo iekšejo text (innerText) ar izveleto burtu masiva (characters[arrayId])
+*/
+
+// 6) izsaucam funkciju getValue()
+
+/* 7) japievino event listener keyPress un parbaudit ievadito to tastaturas vertibu ar izveleto vertibu pirm tam
+        ja gadijuma izvadit paziņojumu alert('PAREIZI') un izsaukt funkciju getValue() vel reizi
+        ne gadijum izvadit paziņojumu alert('NEPAREIZI')
+*/      
+
+const characters = ["a", "c", "e", "g", "i", "k", "m", "o", "r", "t", "v", "z"];
+
+let arrayId;
+let output = document.getElementById("main");
+
+const getValue = () => {
+        arrayId = prompt("Insert a number from 0 to 11!");
+        if (arrayId >= 0 && arrayId <= characters.length -1) {
+               printValue(); 
+        } else {
+                alert("Wrong number or not a number!!!");
+                getValue();
+        }       
+}
+
+const printValue = () => {output.innerText = characters[arrayId]};
+getValue();
+
+document.addEventListener("keypress", (e) => {
+        if (e.key === output.innerText) {
```

Comment:
more logical here would be to use characters[arrayId], you cannot rely on inner text of element.